apps/antalmanac/src/components/Header/LoadSaveFunctionality.tsx

```diff
@@ -32,6 +34,21 @@ interface LoadSaveButtonBaseState {
     rememberMe: boolean;
 }
 
+interface SaveLoadIconProps {
+    loading: boolean;
+    actionName: 'Save' | 'Load';
+}
+
+function SaveLoadIcon(SaveLoadIconProps: SaveLoadIconProps) {
```

Comment:
Generally, prefer to name props literally as `props`. It's easy to understand, and there shouldn't be naming conflicts since functions are distinct entities with different namespaces. The interfaces for the props can be unique to make it clear what component the props are for.

```suggestion
function SaveLoadIcon(props: SaveLoadIconProps) {
```